jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListViewImpl.java

```diff
@@ -15,14 +15,44 @@
  */
 package org.jbpm.workbench.pr.client.editors.instance.list.variables.dash;
 
+import static org.dashbuilder.dataset.filter.FilterFactory.equalsTo;
```

Comment:
It's strange, I imported the three code style definitions into my Eclipse 4.6.2 and the organize imports styles don't seem to work.